python/smashbox/utilities/__init__.py

```diff
@@ -563,13 +563,17 @@ def scrape_log_file(d):
     except AttributeError: # allow this option not to be defined at all
         return
 
-    cmd = 'scp -P %d root@%s:%s/owncloud.log %s/.' % (config.scp_port, config.oc_server, config.oc_server_datadirectory, d)
-    rtn_code,stdout,stderr = runcmd(cmd)
-
-    logger.info('copy command returned %s', rtn_code)
```

Comment:
Does it need an action from my side?

On Thu, Oct 22, 2015 at 4:14 PM, Joas Schilling <notifications@github.com>
wrote:

> In python/smashbox/utilities/__init__.py
> <https://github.com/cernbox/smashbox/pull/35#discussion_r42752748>:
>
> > @@ -563,13 +563,17 @@ def scrape_log_file(d):
> >      except AttributeError: # allow this option not to be defined at all
> >          return
> >
> > -    cmd = 'scp -P %d root@%s:%s/owncloud.log %s/.' % (config.scp_port, config.oc_server, config.oc_server_datadirectory, d)
> > -    rtn_code,stdout,stderr = runcmd(cmd)
> > -
> > -    logger.info('copy command returned %s', rtn_code)
>
> I removed this on merge conflict resolution, will add again
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cernbox/smashbox/pull/35/files#r42752748>.
>



-- 
---
Best regards,
Kuba